### DISCUSSION TITLE
Web UI for Thiruvalluvar GPT

### DISCUSSION THREAD

**Discussion by hariprasd** [codetiger/ThiruvalluvarGPT#3] [Ideas]
Hi sir, I really loved this project and I have an idea to implement this on the web. If you had any ideas we can connect.
I'm a Designer & Front End Developer 
[♪ Hari Prasad](hariprasd.me)

**Comment by codetiger** [comment]
Hi Hariprasad,
Thanks for showing your interest. Yes I've some ideas to extend this to a web interface. I am working on some basic wireframes and can share a basic idea around this. I'll keep you posted here in a few days.

**Reply by hariprasd** [reply]
That's great 👍
